packages/test/src/Happy/Test/SDist.hs

```diff
@@ -0,0 +1,118 @@
+module Happy.Test.SDist(sdist_test) where
```

Comment:
I didn't review this properly, because I'm not familiar with how it used to work. All I know is that you spend quite a lot of time making sure that this is on par with what the Makefile used to do. I have good faith that that's the case.